histFactory/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ else()
 endif()
 
 set(Boost_NO_BOOST_CMAKE ON)
-find_package(Boost REQUIRED)
+find_package(Boost 1.57.0 REQUIRED COMPONENTS system filesystem)
```

Comment:
I changed to 1.46.1 since this is the version installed on Travis, and ingrid has 1.41 (so this hack still works). Of course it's not very robust, so if you have another way of fetching the right libs, that'd good :)